docs/plugins/discogs.rst

```diff
@@ -100,8 +99,7 @@ Here are two things you can try:
 * Make sure that your system clock is accurate. The Discogs servers can reject
   your request if your clock is too out of sync.
 
-The plugin can only match albums, so no Discogs matches will be
-reported when importing singletons using ``-s``. One possible
-workaround is to use the ``--group-albums`` option.
+Support for matching singleton tracks using ``-s`` is in progress.
+If this is not working well, try the ``--group-albums`` option in album import mode.
```

Comment:
I suggest to remove the "in progress" state. Once this PR gets merged I think it will be a grown-up feature and should be stated as that in the docs :-) If new bugs get introduced we need to fix them later on but from my point of view, this feature will probably do it's job as intended.

Also the "is not working well" sounds a little negative. I still think it is very helpful that we mention it as a alternative way of importing files that are not in a album-related folder, so maybe just rephrase.

Reply:
I agree this PR provides mature functionality, but my understanding was that there are other features to add/interfaces to implement before singleton matching is complete. The original staled PR also implemented `track_for_id`, but Adrian asked that `item_candidates` be separated.

I say we list this limitation directly in place of the "in progress" line, something like "matching tracks by ID not yet supported". I'll rephrase the tip as well.